Source/WebKit/NetworkProcess/NetworkSession.h

```diff
@@ -269,6 +271,7 @@ class NetworkSession : public CanMakeWeakPtr<NetworkSession> {
     WebCore::FirstPartyWebsiteDataRemovalMode m_firstPartyWebsiteDataRemovalMode { WebCore::FirstPartyWebsiteDataRemovalMode::AllButCookies };
     WebCore::RegistrableDomain m_standaloneApplicationDomain;
     HashMap<String, WebCore::RegistrableDomain> m_firstPartyHostCNAMEDomains;
+    HashMap<String, String> m_firstPartyHostIPAddresses;
```

Comment:
Do we represent IP addresses as Strings elsewhere? We probably do. Just making sure there's no existing abstraction we should use.

Reply:
Good catch! It looks like `WebCore::IPAddress` is already a thing, so we should definitely use that.